lib/identifier/uuid/uuid.rb

```diff
@@ -1,5 +1,17 @@
 module Identifier
   module UUID
+    MATCHER = %r{
+      [a-f0-9]{8}          # abcdef01
+      -                    #         -
+      [a-f0-9]{4}          #          abcd
+      -                    #              -
+      4[a-f0-9]{3}         #               4abc
+      -                    #                   -
+      [89aAbB][a-f0-9]{3}  #                    8abc
```

Comment:
UUIDs are allowed to be uppercase or lowercase, but only one or the other (I think). So, we could probably get away with insensitivity.